tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
-  "files": ["src/lib/index.d.ts"],
   "compilerOptions": {
     "outDir": "build/",
     "rootDir": ".",
     "lib": ["ES6", "ES2015", "ESNext"],
     "types": ["node", "mocha"],
+    "typeRoots": ["./src/lib/types"],
```

Comment:
Also include node_modules/@types, when you use typeRoots I think it won't read from node_modules

Reply:
or it's working?

Reply:
leave,  ut add node_modules in, it's an array

Reply:
To be fair we have taken this to slack channel, it looks like rootDir is '.' entire project, going to confirm build and if it's all good then we don't need to import typeRoots if it's not outside of rootDir, however the general practise is to have all in src/ dir

Reply:
@mattdean-digicatapult what are your thoughts on the rootDir?